model/src/main/kotlin/utils/LicenseResolver.kt

```diff
@@ -77,10 +80,10 @@ internal class LicenseResolver(private val ortResult: OrtResult) {
 
             // Only license findings of projects can be excluded by path excludes.
             val isExcluded = project != null && finding.locations.all { location ->
-                val path = ortResult.getFilePathRelativeToAnalyzerRoot(project, location.path)
                 pathExcludes.any { exclude ->
-                    exclude.matches(path)
-                        .also { matches -> if (matches) matchingExcludes += exclude }
+                    exclude.matches(location.getRelativePathToRoot()).also { matches ->
```

Comment:
Note: `location.getRelativePathToRoot()` is now called for each iteration of `any` although `location` does not change. Even if the operation is cheap, it's unnecessary.

Reply:
Good point, let me change that.